BUILD

```diff
@@ -254,6 +255,8 @@ GRPCXX_PUBLIC_HDRS = [
     "include/grpcpp/support/time.h",
 ]
 
+grpc_nanopb_library()
```

Comment:
We have a `grpc_generate_one_off_targets` function for exactly this, no need to call it explicitly. Please move it there.

Reply:
Done.